controllers/sr_controller/sr/robot/ruggeduino.py

```diff
@@ -37,9 +37,7 @@ def init_ruggeduino_array(webot):
 
 class Ruggeduino(object):
 
-    DIGITAL_READ_PIN_START = 2  # Exclude pins 0 and 1 as they are used for USB serial comms
-    DIGITAL_WRITE_PIN_START = 7   # Offset output pins by input pins
-    DIGITAL_END_PIN = 12
+    DIGITAL_READ_PIN_START = 2
```

Comment:
Any reason to remove the comment here, or should we add that back?
```suggestion
    DIGITAL_READ_PIN_START = 2  # Exclude pins 0 and 1 as they are used for USB serial comms
```

Reply:
This is also no longer just read, so maybe `DIGITAL_PIN_START`